inst/templates/codemeta-github-actions.yml

```
@@ -0,0 +1,22 @@
+on:
+  push:
+    paths:
+      - DESCRIPTION
+      - .github/workflows/main.yml
+
+name: Render codemeta
+jobs:
+  render:
+    name: Render codemeta
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: r-lib/actions/setup-r@v1
+      - name: Install codemetar
+        run: Rscript -e 'remotes::install.packages("codemetar")'
```

I'm currently testing this. Am I missing something or did you forget a line to install `remotes`? It's present in this repo `main.yml`

---

or 
```suggestion
        run: Rscript -e 'remotes::install_cran("codemetar")'
```

I don't know how caching works for GitHub actions.